hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/infrastructure/providers/ops/BiasedDelegatingProvider.java

```diff
@@ -127,11 +127,20 @@ private void configureDefaults(HapiSpecOperation op) {
         if (shouldAlwaysDefer && isTxnOp) {
             ((HapiTxnOp) op).deferStatusResolution();
         }
+
+        if (op.getPayer().isEmpty()) {
```

Comment:
Not a blocker, just a question; do you know what can cause `op.getPayer()` to be empty?

If the cause is known, we could add a comment above this line explaining the cause.

Reply:
It will be empty if we don't set the payer when we create the operation e.g. `RandomAccountDeletion`, for those cases we default to `UNIQUE_PAYER_ACCOUNT`.
I added this check for the cases when we set the payer to something different e.g. `RandomOperationSignedBy`, so that it will not be overwritten back to `UNIQUE_PAYER_ACCOUNT`.